src/cljs/athens/events.cljs

```diff
@@ -45,7 +45,27 @@
     (let [conn (d/create-conn common-db/schema)]
       (doseq [[_id data] athens-datoms/welcome-events]
         (atomic-resolver/resolve-transact! conn data))
-      {:dispatch [:reset-conn @conn]})))
+      {:async-flow {:id             :db-in-mem-load
```

Comment:
Why async flow here, where it just really does the sync reset conn, but no async flow in `:fs/read-and-watch`?

I think what you did in `:fs/read-and-watch` (just add `:stage/success-db-load` to the dispatch-n) should work here just as well.

Reply:
I've changed `:fs/read-and-watch` to use correct approach too